data/json/items/gun/40mm.json

```diff
@@ -16,12 +41,63 @@
     "range": 30,
     "dispersion": 180,
     "sight_dispersion": 120,
-    "aim_speed" : 6,
+    "aim_speed": 6,
     "recoil": 270,
     "durability": 8,
     "burst": 10,
     "reload": 400,
     "magazines": [ [ "40mm", [ "belt40mm" ] ] ],
     "flags": "MOUNTED_GUN"
+  },
+  {
+    "id": "rm802",
+    "type": "GUN",
```

Comment:
Is copying from the base appropriate here? This weapon's stats are quite different from those of most other launcher-class weapons.